src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -691,6 +691,15 @@ namespace AppInstaller::CLI::Workflow
             return;
         }
 
+        // If the installer claims to have a PackageFamilyName, and that family name is currently registered for the user,
+        // let that be the correlated item and skip any attempt at further ARP correlation.
+        auto installer = context.Get<Execution::Data::Installer>();

```

Comment:
nit const auto& to avoid a copy?

Reply:
The fact that `auto` doesn't preserve the reference in the type constantly escapes my brain, no matter how many times I relearn it...